### DISCUSSION TITLE
Estimate for when will Directus start hiring again for the Software Engineer position?

### DISCUSSION THREAD

**Discussion by daawaan4x** [directus/directus#16684] [Archive]
Hello, I've been eyeing at applying for directus but I never really applied. Just recently learned that the position has been closed and out of curiosity I'd like to ask if is there an estimate for when will Directus start hiring again for the Software Engineer position?

**Comment by wfriedman89** [comment]
@daawaan4U Thank you for your interest in working at Directus.  There isn't a definite time table for positions at this time.  That being said, please continue to monitor our website for job postings.  Closing this thread as this is the Directus Core Code repo.
